addons/web/static/src/core/errors/scss_error_dialog.js

```diff
@@ -13,10 +13,20 @@ ScssErrorDialog.title = _t("Style error");
 const scssErrorDisplayService = {
     dependencies: ["dialog"],
     start(env, { dialog }) {
-        if (window.__odooScssCompilationError) {
-            dialog.add(ScssErrorDialog, {
-                message: window.__odooScssCompilationError,
-            });
+        for (var i = 0; i < document.styleSheets.length; i++) {
+            var styleSheet = document.styleSheets[i];
+            var cssRules = undefined
+            try {
+                cssRules = styleSheet.cssRules;
+            } catch {}
+            if (cssRules && cssRules.length > 0) {
+                var lastRule = cssRules[cssRules.length - 1];
+                if (lastRule.selectorText == 'css_error_message') {
+                    dialog.add(ScssErrorDialog, {
+                        message: lastRule.style['content'].replaceAll('\\a', '\n').replaceAll('\\*', '*').replaceAll('\\"', '"'),
+                    });
+                }
+            }
```

Comment:
```suggestion
        const assets = [...document.styleSheets].filter((sheet) => sheet.href?.startsWith("/web/assets"));
        for (const { cssRules } of assets) {
            const lastRule = cssRules?.[cssRules?.length - 1];
            if (lastRule?.selectorText === "css_error_message") {
                dialog.add(ScssErrorDialog, {
                    message: lastRule.style.content.replaceAll("\\a", "\n").replaceAll("\\*", "*").replaceAll(`\\"`, `"`),
                });
            }
```
Maybe something like this?